index.html

```diff
@@ -51,29 +51,22 @@ <h2>Archive:</h2>
         twttr.widgets.createTweet(archiveTweetId, tweetDiv, {});
       });
 
+      // Use latest tweet id to get the timestamp
+      var date = new Date(Math.floor(String(Math.floor(latestTweetId / 4194304))) + 1288834974657);
+
+      // Subtract from now and convert to days (from milliseconds)
+      var daysAgo = (new Date - date) / 1000 / 60 / 60 / 24;
```

Comment:
since you're working with ints now, why use `Date` at all? :)

Reply:
How else can I get the current timestamp in milliseconds?

Reply:
`Date.now()` seems OK
also, subtracting a number from a `Date` would cast the `Date` to a number (so it would return `ms`) :)

Reply:
so, this should do just fine:

```js
// Use latest tweet id to get the timestamp
var date = Math.floor(latestTweetId / 4194304) + 1288834974657;

// Subtract from now and convert to days (from milliseconds)
var daysAgo = (Date.now() - date) / 1000 / 60 / 60 / 24;
```

Reply:
@stryju: if you look at master you'll see that after merging in @perryizgr8's change I essentially made your suggestions in a subsequent commit